src/zope/component/tests/test_hookable.py

```diff
@@ -68,62 +68,38 @@ def bar():
     def test_cant_assign_original(self):
         from zope.component.hookable import hookable
         def foo():
-            pass
+            raise NotImplementedError()
```

Comment:
@tseaver I updated the PR to use the `foo = fails_if_called()` approach instead of `NotImplementedError`. You're right, that does a better job capturing the intent and even reads easier, to me at least.